_pages/invite-letter.html

```diff
@@ -0,0 +1,98 @@
+---
+layout: base
+title: Visa Invitation Letter
+heading: Visa Invitation Letter
+permalink: /invitation/
+redirect_from:
+- /about/organizers/
```

Comment:
```suggestion
permalink: /invitation/
```

I don't think we need this redirect, right?